.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+  "recommendations": [
```

Comment:
nice!!! 🎉 🌮 

This will be super helpful for getting everyone on the same linting tools.